### PR TITLE
Add consent API endpoints

### PR DESCRIPTION
**Pull request by Francois-Gomis**

### CODE REVIEW

**Review by syjust-alma — COMMENTED** [review]
thank you for the feature.
everything is ok but I would wait for the #8 finalisation to add this new endpoint with the new architecture of the php-client

File: src/Endpoints/ShareOfCheckout.php [hunk 1]
```diff
@@ -35,4 +35,29 @@ public function getLastUpdateDates()
         return $res->json;
     }
 
+    /**
+     * Consent share of checkout
+     * @throws RequestError
+     */
```
Comment (ClaireBonacheraAlma) [inline src/Endpoints/ShareOfCheckout.php:41]: missing @return

File: src/Endpoints/ShareOfCheckout.php [hunk 2]
```diff
@@ -35,4 +35,29 @@ public function getLastUpdateDates()
...
+        $res = $this->request(self::SHARE_OF_CHECKOUT_PATH . 'consent')->setRequestBody()->post();
+        if ($res->isError()) {
+            throw new RequestError($res->errorMessage, null, $res);
+        }
+        return $res->json;
+    }
+
+    /**
+     * Consent share of checkout
+     * @throws RequestError
```
Comment (ClaireBonacheraAlma) [inline src/Endpoints/ShareOfCheckout.php:53]: missing @return

**Review by ClaireBonacheraAlma — APPROVED** [review]

**Review by syjust-alma — APPROVED** [review]
PR #8 takes to many times.

We will see how to refactor this feature later.

Ok for me. Thank you for the job 🙏

**Pull request merged**